TradingBotTests/manual-tests.http

```diff
@@ -18,6 +18,16 @@ POST http://localhost:5000/manual-tests/trading-actions
 Authorization: Basic YWRtaW46cGFzc3dvcmQ=
 Content-Type: application/json
 
+### Strategy results execution
+GET http://localhost:5000/manual-tests/strategy-results
+Authorization: Basic YWRtaW46cGFzc3dvcmQ=
+Content-Type: application/json
+
+### Trading cycle execution
+GET http://localhost:5000/manual-tests/trading-cycle-test
+Authorization: Basic YWRtaW46cGFzc3dvcmQ=
+Content-Type: application/json
+
```

Comment:
The JSON below this is a request sent in POST request above this. So these requests should probably be below this JSON (or above the POST request).